src/app/Http/Controllers/API/PresentationController.php

```diff
@@ -68,7 +68,7 @@ public function store(Request $request, User $user)
      */
     public function show(Presentation $presentation)
     {
-        //
+        return new PresentationResource($presentation);
     }
 
     /**
```

Comment:
This file should not be present on your PR

Reply:
those are leftover from the blade.php misadventure

Reply:
after a second check, thoses files are still on my branch, but I believe removing them will cause huge problems when we merge.